app/views/users/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
+  </div>
+<% end %>
+
+<%= render "users/shared/links" %>
```

Comment:
- [ ] And when I ask for confirmation of email, by clicking the `Didn't receive confirmation instructions? button I'm getting this browser error:
👍🏻 

![image](https://user-images.githubusercontent.com/79058364/224463662-993d6375-ec22-4ae7-8d2c-19e816678eca.png)

- And my console:

![image](https://user-images.githubusercontent.com/79058364/224463692-058e02bd-fd20-4b8d-ad6a-8f24a0a96542.png)


- Kindly make sure that your web app that enables the registration, confirmable, and login of users.